var/spack/repos/builtin/packages/py-sphinxcontrib-jquery/package.py

```diff
@@ -12,8 +12,11 @@ class PySphinxcontribJquery(PythonPackage):
     homepage = "https://github.com/sphinx-contrib/jquery"
     pypi = "sphinxcontrib-jquery/sphinxcontrib-jquery-2.0.0.tar.gz"
 
+    version("4.1", sha256="1620739f04e36a2c779f1a131a2dfd49b2fd07351bf1968ced074365933abc7a")
     version("2.0.0", sha256="8fb65f6dba84bf7bcd1aea1f02ab3955ac34611d838bcc95d4983b805b234daa")
 
-    depends_on("python@3.5:", when="@2:", type=("build", "run"))
-    depends_on("py-setuptools@65:", type=("build", "run"))
     depends_on("py-flit-core@3.7:", when="@3:", type="build")
+    depends_on("py-sphinx@1.8:", type=("build", "run"))
+
+    # Historical dependencies
+    depends_on("py-setuptools@65:", when="@:3", type=("build", "run"))
```

Comment:
Any idea where 65+ comes from?

Reply:
Not really, but I suspect that the one who added it wanted to add some restriction to make sure that `setuptools` can handle installation via `pyproject.toml`

Reply:
Oh, it's right there in the `pyproject.toml`, not sure how I missed that. Thanks!